packages/components/Switch/src/SwitchTokens.ts

```diff
@@ -22,6 +22,8 @@ export const defaultSwitchTokens: TokenSettings<SwitchTokens, Theme> = (t: Theme
   trackMarginLeft: 2,
   trackMarginRight: 2,
   thumbMargin: 2,
+  onOpacity: 0,
```

Comment:
talked offline about using absolute positioning to stack the two texts and use opacity, avoid layout change possibilities from the height changing